src/RemoteInboxNotifications/DataSourcePoller.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Handles polling and storage of specs
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications;
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\WooCommerce\Admin\Loader;
+
+/**
+ * Specs data source poller class.
+ * This handles polling specs from JSON endpoints, and
+ * stores the specs in to the database as an option.
+ */
+class DataSourcePoller {
```

Comment:
As this class only has static members, I think just namespacing the functions would be better.

Reply:
I agree, I didn't know you could namespace functions in PHP. I've tried this and couldn't get it to work. The `DataSourcePoller.cs` file looks like:
```
namespace Automattic\WooCommerce\Admin\RemoteInboxNotifications;

function read_specs() {
    ...
}
```
and I'm using it like:
```
use function \Automattic\WooCommerce\Admin\RemoteInboxNotifications\read_specs;
...
read_specs();
```
and I'm getting this error:
```
[19-May-2020 14:57:25 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function Automattic\WooCommerce\Admin\RemoteInboxNotifications\read_specs() in /srv/www/wordpress-two/public_html/wp-content/plugins/woocommerce-admin/src/Events.php:82
```

I can't find any usages of `use function` across wc-admin and wc-core so I can't see if I'm using the namespaces correctly. Do you know why this wouldn't work?

Reply:
Hmm... odd. That looks like it should work to me, but admittedly I do not consider myself an advanced PHP developer, so perhaps I'm overlooking a crucial detail in regards to namespacing functions.